dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -195,7 +195,7 @@ ANNO_CLASSINFO_EXISTS4.useraction=No user action is available.
 # # {1} the name of a class
 ANNO_CLASSINFO_SCAN_EXCEPTION=CWWKC0022W: [ {0} ] The scan of class [{1}] caused an exception. The message is: [{2}] caused by [{3}].
 ANNO_CLASSINFO_SCAN_EXCEPTION.explanation=The scan of a class for annotations caused an exception.
-ANNO_CLASSINFO_SCAN_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSINFO_SCAN_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.